scripts/cloudScript.js

```diff
@@ -1318,14 +1318,18 @@ function loadData(id,kid) {
         if (data.city.halloween === undefined) {
```

Comment:
We should reset the event for people who played it last year ^^

```suggestion
        if (data.city.halloween === undefined || data.city.halloween.dailyClaimed < 18561) {
```